cli/gluster-block.c

```diff
@@ -259,6 +271,27 @@ glusterBlockIsNameAcceptable(char *name)
   return TRUE;
 }
 
+static bool
+glusterBlockIsVolListAcceptable(char *name)
+{
+  char *end, *tok, *tmp;
+  char delim[2] = {'\0', };
+
+
+  if (!name || GB_STRDUP(end, name) < 0)
+    return FALSE;
+
+  tmp = end;
+  delim[0] = GB_VOLS_DELIMITER;
+  while ((tok = strsep(&end, delim))) {
```

Comment:
As per manpage, strtok is more portable:
NOTES
       The strsep() function was introduced as a replacement for strtok(3), since the latter cannot handle empty fields.  However, strtok(3) conforms to C89/C99 and hence is more portable.